doc/modules/model_evaluation.rst

```diff
@@ -1033,6 +1033,60 @@ set [0,1] has an error: ::
     for an example of zero one loss usage to perform recursive feature
     elimination with cross-validation.
 
+.. _brier_score_loss:
+
+Brier score loss
+----------------
+
+The :func:`brier_score_loss` function computes the
+`Brier score <https://en.wikipedia.org/wiki/Brier_score>`_
+for binary classes. Quoting Wikipedia:
+
+   "The Brier score is a proper score function that measures the accuracy of
+    probabilistic predictions. It is applicable to tasks in which predictions
+    must assign probabilities to a set of mutually exclusive discrete outcomes.
+    The set of possible outcomes can be either binary or categorical in nature,
+    and the probabilities assigned to this set of outcomes must sum to one
+    (where each individual probability is in the range of 0 to 1). It was proposed
+    by Glenn W. Brier in 1950."
+
+This function returns a score of the mean square difference between the actual outcome and
+the predicted probability of the possible outcome. The actual outcome has to be 1 or 0
+(true or false), while the predicted probability of the actual outcome can be
+a value between 0 and 1.
+The brier score loss is also between 0 to 1 and the lower the score
+(the mean square difference is smaller), the more accurate the prediction is.
+It can be thought of as a measure of the "calibration" of a set of probabilistic predictions.
+`Here <ttp://docs.lib.noaa.gov/rescue/mwr/078/mwr-078-01-0001.pdf>`_ is
+the paper describing it.
```

Comment:
Instead of putting an inline link with "Here", please use the standard "Reference" section format used consistently throughout the scikit-learn documentation.